troposphere/template_generator.py

```diff
@@ -307,10 +308,11 @@ def _get_function_type(self, function_name):
 
     def _import_all_troposphere_modules(self):
         """ Imports all troposphere modules and returns them """
+        dirname = os.path.join(os.path.dirname(__file__))
         module_names = [
             pkg_name
             for importer, pkg_name, is_pkg in
-            pkgutil.walk_packages(__file__)
+            pkgutil.walk_packages([dirname], prefix="troposphere.")
```

Comment:
Ah, good point; Yes I believe that this can now be removed; I'll try to update this pr later tonight.